src/components/styles.js

```diff
@@ -0,0 +1,201 @@
+import { StyleSheet, Dimensions } from "react-native";
+/**
+ * This is the style sheet for the log in/sign up forms.
+ */
+const styles = StyleSheet.create({
+  textForgotPass: {
+    fontSize: 19,
+    fontWeight: "bold",
+    letterSpacing: 0.25,
+    color: "darkviolet",
+  },
+  buttonForgotPass: {
+    alignItems: "center",
+    justifyContent: "center",
+    borderColor: "darkviolet",
+    borderWidth: 1,
+    backgroundColor: "white",
+    borderRadius: 100,
+    marginTop: 10,
```

Comment:
also, any margin or padding should not be in a general style but in a component-specific style